src/Interpreters/RowRefs.cpp

```diff
@@ -138,7 +104,6 @@ std::optional<TypeIndex> AsofRowRefs::getTypeSize(const IColumn & asof_column, s
         case TypeIndex::Int64:
             size = sizeof(Int64);
             return idx;
-        //case TypeIndex::Int128:
```

Comment:
Need to investigate why big integers are not supported.

Reply:
It was added like that in the PR adding types (https://github.com/ClickHouse/ClickHouse/pull/11301/files) but I don't see any reason as to why big integers shouldn't be supported. Specially since `Decimal128` is supported. I'll try to enable them and see what happens.